### PR TITLE
Feature/upmcfhir 346 jf device authorship

### PR DESCRIPTION
**Pull request by jmfadeley**
#### What does this PR do?
Adds the assigned author device to the CDA elements. For now, just inspect the code and tell me what I'm missing.

### CODE REVIEW

File: src/main/java/tr/com/srdc/cda2fhir/transform/ResourceTransformerImpl.java [hunk 1]
```diff
@@ -553,6 +554,64 @@ public EntryResult tAllergyProblemAct2AllergyIntolerance(AllergyProblemAct cdaAl
 		return result;
 	}
 
+	@Override
+	public EntityResult tAssignedAuthor2Device(AssignedAuthor cdaAssignedAuthor, IBundleInfo bundleInfo) {
+		if (cdaAssignedAuthor == null || cdaAssignedAuthor.isSetNullFlavor()) {
+			return new EntityResult();
```
Comment (mmccall) [inline src/main/java/tr/com/srdc/cda2fhir/transform/ResourceTransformerImpl.java:563]: you might be able to just return null here, not sure though.

File: src/main/java/tr/com/srdc/cda2fhir/transform/ResourceTransformerImpl.java [hunk 2]
```diff
@@ -553,6 +554,64 @@ public EntryResult tAllergyProblemAct2AllergyIntolerance(AllergyProblemAct cdaAl
...
+			fhirDevice.setManufacturer(
+					cdaAssignedAuthor.getAssignedAuthoringDevice().getManufacturerModelName().getDisplayName());
+			fhirDevice.setVersion(cdaAssignedAuthor.getAssignedAuthoringDevice().getSoftwareName().getText());
+		}
+
+		// representedOrganization -> EntityInfo.organization
+		if (cdaAssignedAuthor.getRepresentedOrganization() != null
+				&& !cdaAssignedAuthor.getRepresentedOrganization().isSetNullFlavor()) {
+			Organization fhirOrganization = tOrganization2Organization(cdaAssignedAuthor.getRepresentedOrganization());
+			info.setOrganization(fhirOrganization);
```
Comment (mmccall) [inline src/main/java/tr/com/srdc/cda2fhir/transform/ResourceTransformerImpl.java:631]: I'd just double check that this org is getting added to the bundle.

**Review by mmccall — COMMENTED** [review]
Generally seems correct, guess you just need to test. Also would think Jolt would get upset about Device, but 🤷‍♂️.

**Comment by jmfadeley** [comment]
@mmccall Tested and fixed I think. Still awaiting Jolt.

**Comment by jmfadeley** [comment]
@mmccall I've added the unit test. I would note however, that the test is supposed to be writing the bundles to a `.txt` file. Not just mine, but none of them are. But I've used the debugger to confirm that the bundle _is_ created.

**Review by mmccall — APPROVED** [review]

**Pull request merged**